pgcli/pgclirc

```diff
@@ -135,6 +132,7 @@ less_chatty = False
 # \H - Hostname of the server
 # \d - Database name
 # \p - Database port
+# \e - Environment name
```

Comment:
Not going to add environment name anymore, right?

Reply:
Missed that, I'll remove it.